perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -223,19 +224,27 @@ sub updates_as_hashref {
     return $hashref;
 }
 
+# If $num is undefined, we want to return the minimum photo number that can be
+# shown (1-indexed), or false for no display. If $num is defined, return
```

Comment:
This says it's 1-indexed but unless I've misread something the keys are going to be 0-indexed, based on [`loop.index` here](https://github.com/mysociety/fixmystreet/pull/1966/commits/fc752cd809606bf9b59746878d73d23265f7d93d#diff-e67fb616e00fc667a303d5ddfcba2623R121) and where it's used [here](https://github.com/mysociety/fixmystreet/pull/1966/commits/fc752cd809606bf9b59746878d73d23265f7d93d#diff-9a83c02c2deb49dcad095e248b48a878R597).

Reply:
Oh yes, I need to add 1 to the `return min grep...` line so that it doesn't return 0 (failure). Will add that test for this!